src/DependencyManager.ts

```diff
@@ -29,11 +29,12 @@ interface PackageJson {
 }
 
 export class DependencyManager {
-	private json: PackageJson;
+	private json!: PackageJson;
 	path: string;
 
 	constructor(path: string = resolve(process.cwd(), 'package.json')) {
-		this.setPackagePath(path);
+		this.path = path;
```

Comment:
The latest TypeScript complained about member variables not being initialized. `setPackagePath` wasn't used externally (aside from tests). Moved into the constructor.